### PR TITLE
Remove KDE mentions & Reorganize images in the "Choosing your distro" post

### PR DESCRIPTION
**Pull request by TommyTran732**
Only GNOME should be recommended.

### CODE REVIEW

File: content/posts/linux/Choosing Your Desktop Linux Distribution/index.md [hunk 1]
```diff
@@ -60,31 +60,29 @@ There is often some confusion about “security-focused” distributions and “
...
+## Desktop Environments
 
-You should use a desktop environment that supports the [Wayland](https://en.wikipedia.org/wiki/Wayland_(display_server_protocol)) display protocol as it developed with security [in mind](https://lwn.net/Articles/589147/). Its predecessor, [X11](https://en.wikipedia.org/wiki/X_Window_System), does not support GUI isolation, allowing all windows to [record screen, log and inject inputs in other windows](https://blog.invisiblethings.org/2011/04/23/linux-security-circus-on-gui-isolation.html), making any attempt at sandboxing futile. While there are options to do nested X11 such as [Xpra](https://en.wikipedia.org/wiki/Xpra) or [Xephyr](https://en.wikipedia.org/wiki/Xephyr), they often come with negative performance consequences, are not convenient to set up, and are not preferable to Wayland.
+You should use GNOME as your desktop environment. It supports [Wayland](https://en.wikipedia.org/wiki/Wayland_(display_server_protocol)) and implements permission control for privileged Wayland protocols like screencopy. There are other desktop environments and window managers with Wayland support, but I am not aware of any permission control implemeneted by them.
```
Comment (friendly-rabbit-35) [inline content/posts/linux/Choosing Your Desktop Linux Distribution/index.md:65]:
```suggestion
You should use GNOME as your desktop environment. It supports [Wayland](https://en.wikipedia.org/wiki/Wayland_(display_server_protocol)), a display protocol developed with security [in mind](https://lwn.net/Articles/589147), and implements permission control for privileged Wayland protocols like screencopy. There are other desktop environments and window managers with Wayland support, but I am not aware of any permission control implemented by them.
```
I also suggest embedding a link to a relevant merge request or issue about the permission control mentioned here.

Not suggesting this link necessarily, but just something I found that might be relevant: https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1970

File: content/posts/linux/Choosing Your Desktop Linux Distribution/index.md [hunk 2]
```diff
@@ -60,31 +60,29 @@ There is often some confusion about “security-focused” distributions and “
...
-
-Try **not** to use desktop environments or window managers that do not have Wayland support such as Cinnamon (default on Linux Mint), Pantheon (default on Elementary OS), MATE, Xfce, and i3. You should also avoid Wayland environments which use wlroots as the compositor such as Sway, since wlroots adds many of X11 weaknesses back, including the ability for apps to record each other without prompting for permission.
+Wayland's predecessor, [X11](https://en.wikipedia.org/wiki/X_Window_System), does not support GUI isolation, allowing all windows to [record screen, log and inject inputs in other windows](https://blog.invisiblethings.org/2011/04/23/linux-security-circus-on-gui-isolation.html), making any attempt at sandboxing futile. While there are options to do nested X11 such as [Xpra](https://en.wikipedia.org/wiki/Xpra) or [Xephyr](https://en.wikipedia.org/wiki/Xephyr), they often come with negative performance consequences, are not convenient to set up, and are not preferable to Wayland. Desktop environments and window managers which only support X11 should be avoided.
```
Comment (friendly-rabbit-35) [inline content/posts/linux/Choosing Your Desktop Linux Distribution/index.md:67]:
```suggestion
Wayland's predecessor, [X11](https://en.wikipedia.org/wiki/X_Window_System), does not support GUI isolation, allowing all windows to [record screen, log and inject inputs in other windows](https://blog.invisiblethings.org/2011/04/23/linux-security-circus-on-gui-isolation.html), making any attempt at sandboxing futile. While there are options to run nested X11 sessions such as [Xpra](https://en.wikipedia.org/wiki/Xpra) or [Xephyr](https://en.wikipedia.org/wiki/Xephyr), they often come with negative performance consequences, are not convenient to set up, and are not preferable to Wayland. You should avoid desktop environments and window managers which only support X11.
```

File: content/posts/linux/Choosing Your Desktop Linux Distribution/index.md [hunk 3]
```diff
@@ -60,31 +60,29 @@ There is often some confusion about “security-focused” distributions and “
...
 
-### Fedora Silverblue & Kinoite
+### Fedora Silverblue
 
-[Fedora Silverblue](https://silverblue.fedoraproject.org/) and [Fedora Kinoite](https://kinoite.fedoraproject.org/) are immutable variants of Fedora with a strong focus on container workflows. Silverblue comes with the [GNOME](https://www.gnome.org/) desktop environment while Kinoite comes with [KDE](https://kde.org/). Silverblue and Kinoite follow the same release schedule as Fedora Workstation, benefiting from the same fast updates and staying very close to upstream.
+[Fedora Silverblue](https://silverblue.fedoraproject.org/) is an immutable variants of Fedora with a strong focus on container workflows. It follows the same release schedule as Fedora Workstation, benefiting from the same fast updates and staying very close to upstream.
```
Comment (friendly-rabbit-35) [inline content/posts/linux/Choosing Your Desktop Linux Distribution/index.md:85]:
```suggestion
[Fedora Silverblue](https://silverblue.fedoraproject.org/) is an immutable variant of Fedora with a strong focus on container workflows. It follows the same release schedule as Fedora Workstation, benefiting from the same fast updates and staying very close to upstream.
```

**Review by friendly-rabbit-35 — CHANGES_REQUESTED** [review]

**Comment by MorsMortium** [comment]
What kind of permission control?
KDE wayland with pipewire has these:
Screenshot_20240730_201654
Screenshot_20240730_201713
Without the user clicking them, the app can't access the screen/window.

**Comment by TommyTran732** [comment]
@MorsMortium

Can you check with Flameshot? I am pretty sure @qoijjj tested it and there isn't any prompt

**Comment by qoijjj** [comment]
@TommyTran732 when I tested it, I was able to capture the screen by layering `flameshot` on kinoite, yes. With no prompt

what @MorsMortium shared looks like the wayland screenshare popup, which as far as I know is unrelated to the wl screencopy protocol

**Comment by qoijjj** [comment]
@MorsMortium if you are unable to repro, that would be very interesting

**Comment by MorsMortium** [comment]
Yeah, these are for screen sharing. Didn't know screen copy was something else. Both spectacle and flameshot can make screenshots

**Comment by jermanuts** [comment]
https://discuss.privacyguides.net/t/people-should-avoid-wayland-environments-which-use-wlroots-as-the-compositor/17899/8

**Comment by qoijjj** [comment]
@jermanuts I responded on the thread

**Pull request merged**